ministryofjustice/correspondence_tool_staff
# CT-3143 Case creation for offender SAR complaint

## Description
This PR implements the case creation flow and display (show page) for Offender SAR complaint, including the controller, views, site prism files and so on 

It also adds a new task to run the correspondence type seeder in use; this should be safe to run on production environments as it does find or create but please pay extra attention to this to verify no unwanted side effects.

## Self-review checklist


### Screenshots

![image](https://user-images.githubusercontent.com/1161161/95721413-75f52980-0c6a-11eb-9056-56ae959f551e.png)


### Related JIRA tickets
https://dsdmoj.atlassian.net/browse/CT-3143

### Deployment
Run the correspondence type seeder (and restart the app pods? @ymao2 is this right?)

`rails db:seed:dev:correspondence_types`

### Manual testing instructions

Create a new Offender SAR case and verify that the details display as desired on the Case Show page
